tests/system/test_root_metronome.py

```diff
@@ -280,4 +279,7 @@ def job(job_json):
     try:
         yield
     finally:
-        client.remove_job(job_id, True)
+        try:
```

Comment:
this took me ~1h to figure out :D because the test was still not giving me the nice assert failures and it was because when this thing failed, it printed the dcos error and not the nice assertion error...

Reply:
btw. we do the same thing with exceptions in Marathon cleanup methods

Reply:
You mean it printed the error of the cleanup? Good catch!